src/binaries/firmware/bin/download.sh

```diff
@@ -40,7 +40,7 @@ cd "$BIN_DIR"
 
 # mark as executable and patch for Nix
 chmod u+x trezor-emu-*
-nix-shell -p autoPatchelfHook SDL2 SDL2_image --run "autoPatchelf trezor-emu-*"
+nix-shell -p autoPatchelfHook SDL2 SDL2_image -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/7fad01d9d5a3f82081c00fb57918d64145dc904c.tar.gz --run "autoPatchelf trezor-emu-*"
```

Comment:
Perhaps you can set `NIX_PATH` variable in shell.nix? Should work the same as `-I`.

Reply:
Yup, this seems to work! Added in https://github.com/trezor/trezor-user-env/pull/119/commits/5e3894ca30015c21fc81ab5fdfe1a8276a77307e.